src/remote/activitypub/kernel/delete/actor.ts

```diff
@@ -0,0 +1,26 @@
+import { apLogger } from '../../logger';
+import { createDeleteAccountJob } from '@/queue';
+import { IRemoteUser } from '@/models/entities/user';
+import { Users } from '@/models/index';
+
+const logger = apLogger;
+
+export async function deleteActor(actor: IRemoteUser, uri: string): Promise<string> {
+	logger.info(`Deleting the Actor: ${uri}`);
+
+	if (actor.uri !== uri) {
+		return `skip: delete actor ${actor.uri} !== ${uri}`;
+	}
+
+	if (actor.isDeleted) {
+		logger.info(`skip: already deleted`);
+	}
+
+	const job = await createDeleteAccountJob(actor);
```

Comment:
The second parameter in this function call is missing, causing this error message `[queue.inbox] failed(TypeError: Cannot read property 'soft' of undefined)` and jobs being stuck on the inbox queue.

The solution with the least modification would probably be

```suggestion
	const job = await createDeleteAccountJob(actor, {});
```

Or the function definition in `queue.ts` could be adjusted like this:
```
export function createDeleteAccountJob(user: ThinUser, opts: { soft?: boolean; } = {}) {
```

Reply:
いや、呼び出し先がBREAKING CHANGEしてるから呼び出し先を治すべきでしょ

Reply:
@syuilo

Reply:
https://github.com/misskey-dev/misskey/pull/7830